fix(ui): pin ssi agent validate button to bottom of page

Sotatek-DukeVu opened the pull request:
## Description

Pin validate button to bottom of `Create SSI agent` page

## Checklist before requesting a review

### Issue ticket number and link

- [x] This PR has a valid ticket number or issue: [DTIS-971](https://cardanofoundation.atlassian.net/browse/DTIS-971)

### Testing & Validation

- [x] This PR has been tested/validated in IOS, Android and browser.
- [x] The code has been tested locally with test coverage match expectations.
- [] Added new Unit/Component testing (if relevant).

### Security

- [x] No secrets are being committed (i.e. credentials, PII)
- [x] This PR does not have any significant security implications

### Code Review

- [x] There is no unused functionality or blocks of commented out code (otherwise, please explain below)
- [x] In addition to this PR, all relevant documentation (e.g. Confluence) and architecture diagrams (e.g. Miro) were updated

### Design Review

- [x] If this PR contains changes to the UI, it has gone through a design review with UX Designer or Product owner.
- [x] In case PR contains changes to the UI, add some screenshots to notice the differences
---
#### IOS
<img width="1106" alt="Screenshot 2024-06-14 at 15 56 36" src="https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/e41d0976-2d03-4f91-b294-af69bf07d110">

#### Android
![Screenshot 2024-06-14 at 15 28 23](https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/556fee54-77ff-4288-ae59-57c6f59fb7bb)

#### Browser
![Screenshot 2024-06-14 at 15 22 39](https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/0849afb5-63e9-4bb2-94da-fca2a49686fa)


[DTIS-971]: https://cardanofoundation.atlassian.net/browse/DTIS-971?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

sdisalvo-crd reviewed (commented):
Could you please also fix “Create your password” page, I added a screenshot to the ticket on Jira to show it has the same issue.

Sotatek-DukeVu commented:
> Could you please also fix “Create your password” page, I added a screenshot to the ticket on Jira to show it has the same issue.

I updated create password screen. Please help me check.

#### IOS
<img width="1111" alt="Screenshot 2024-06-17 at 09 47 33" src="https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/6a8cfb24-ba31-4255-a5da-a7a3dd3098c9">


#### Android
![Screenshot 2024-06-17 at 09 55 40](https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/fe574355-913f-4f3d-b5f9-fbb07061f417)


#### Browser
![Screenshot 2024-06-17 at 09 35 48](https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/210f4840-2608-4bcb-9dab-668910d4bd15)
